src/websocket_server/server.py

```
@@ -24,39 +24,39 @@ class _State:
-        global _thread
+def sender_loop():
+    global _thread
+    MAX_RESTARTS = 5
+    for attempt in range(MAX_RESTARTS):
         try:
             sender = Sender()
             while True:
                 msg = _relay_queue.get()
                 sender.send_message(msg)
                 socketio.sleep(0)  # Yield to the SocketIO event loop to prevent blocking
         except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)